configs/facades.py

```diff
@@ -0,0 +1,26 @@
+import ml_collections
+
+from configs import commons
+
+
+def get_config() -> ml_collections.ConfigDict:
+    config = ml_collections.ConfigDict()
+
+    config.common_configs = commons.get_config()
```

Comment:
Also, might be a good idea to add a path for model serialization and intermediate result dumps.